app/components/grouped_cards/listing_component.rb

```diff
@@ -0,0 +1,19 @@
+module GroupedCards
+  class ListingComponent < ViewComponent::Base
+    def initialize(data)
+      @data = data
+    end
+
+    def regions
+      @data.keys
+    end
+
+    def region_link_anchor(region)
+      region.downcase.gsub %r{[^a-z0-9]+}, "-"
```

Comment:
Would `parameterize` work here instead of a regex? you'd get a slightly different result in that it omits symbols, but I'm not sure if that would matter?

```
region.parameterize(separator: "-")
```

Reply:
Hah - good call, I'd forgotten about that one - I'll see if it behaves as I want